addons/nametags/functions/fnc_drawNameTagIcon.sqf

```diff
@@ -57,10 +57,10 @@ _fnc_parameters = {
     private _color = [1, 1, 1, _alpha];
     if ((group _target) != (group _player)) then {
         _color = +GVAR(defaultNametagColor); //Make a copy, then multiply both alpha values (allows client to decrease alpha in settings)
-        _color set [3, (_color select 3) * _alpha];
     } else {
-        _color = [[1, 1, 1, _alpha], [1, 0, 0, _alpha], [0, 1, 0, _alpha], [0, 0, 1, _alpha], [1, 1, 0, _alpha]] select ((["MAIN", "RED", "GREEN", "BLUE", "YELLOW"] find ([assignedTeam _target] param [0, "MAIN"])) max 0);
+        _color = [+GVAR(nametagColorMain), +GVAR(nametagColorRed), +GVAR(nametagColorGreen), +GVAR(nametagColorBlue), +GVAR(nametagColorYellow)] select ((["MAIN", "RED", "GREEN", "BLUE", "YELLOW"] find ([assignedTeam _target] param [0, "MAIN"])) max 0);
```

Comment:
Should be optimized to:
```sqf
_color = + ([GVAR(nametagColorMain), GVAR(nametagColorRed), GVAR(nametagColorGreen), GVAR(nametagColorBlue), GVAR(nametagColorYellow)] select ((["MAIN", "RED", "GREEN", "BLUE", "YELLOW"] find ([assignedTeam _target] param [0, "MAIN"])) max 0));
```

Reply:
No need to copy 5 arrays every draw frame, when you could just copy the chosen array, whichever it is.

Reply:
That's a good point, will do